### ISSUE TITLE
cf.skipSslValidation not working

### ISSUE THREAD

**Issue by ntrip** [promregator/promregator#79]
Hi Team,

**Problem Statement:**

I am trying to run Promregator basic setup by following this:
https://github.com/promregator/promregator/blob/master/docs/quickstart-docker.md

but I am getting below error
`Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target`

I got cf.skipSslValidation config https://github.com/promregator/promregator/blob/master/docs/config.md which I think I should set, but I couldn't set it properly. I have tried multiple values for skipSslValidation in the promregator.yml file but getting same error:

```
cf:
  api_host: api.cf.example.org
  username: myCFUserName
  skipSslValidation: true / TRUE / enable / ENABLE / none / NONE / yes / YES
```

**Environment:**
Promregator Application v0.1.0 
OS: Ubuntu 18.04 VM

**Below are my questions:**
1) How can I skip the SSL validation for Promregator demo purpose?
2) If I have the SSL Key file then where should I add it in Ubuntu 18.04? I have tried to put my key in this file /etc/ssl/certs/ca-certificates.crt but got the same error.

Thanks in advance!

Regards,
Neetu

**Comment by eaglerainbow** [comment]
Hi Neetu,

thanks for checking out Promregator.
On your request I'd have several questions:

* With your link to the config documentation, you are referring to the "most recent" version of Promregator ("master" branch). It reflects the state which will become active once version 0.5.0 will be released. The most recent released version is [0.4.1](https://github.com/promregator/promregator/releases/tag/v0.4.1). Are you aware of this difference?
* You say that you are still using 0.1.0 - which is ... ancient. Is there a certain reason why you want to use such a historical version anymore?
* `cf.skipSslValidation` was introduced with 38a85b79ab430ba7f31c0b49d106179e7fa644ef and only got available with `0.2.0`. Is there a reason why you expect it to see in 0.1.0?

Hope this helps a little!

**Comment by eaglerainbow** [comment]
> If I have the SSL Key file then where should I add it in Ubuntu 18.04? I have tried to put my key in this file /etc/ssl/certs/ca-certificates.crt but got the same error.

BTW: Did you also run [`sudo update-ca-certificates`](http://manpages.ubuntu.com/manpages/xenial/man8/update-ca-certificates.8.html)?

**Comment by ntrip** [comment]
Hi Nico,

Thanks a ton for your quick response!

> * With your link to the config documentation, you are referring to the "most recent" version of Promregator ("master" branch). It reflects the state which will become active once version 0.5.0 will be released. The most recent released version is [0.4.1](https://github.com/promregator/promregator/releases/tag/v0.4.1). Are you aware of this difference?
Oh yeah, I was referring to the latest one. Will check the differences. Thanks!

> * You say that you are still using 0.1.0 - which is ... ancient. Is there a certain reason why you want to use such a historical version anymore?
I was following this [document](https://github.com/promregator/promregator/blob/master/docs/quickstart-docker.md) so I was using 0.1.0. I will definitely try the 0.4.1 version.

> * `cf.skipSslValidation` was introduced with [38a85b7](https://github.com/promregator/promregator/commit/38a85b79ab430ba7f31c0b49d106179e7fa644ef) and only got available with `0.2.0`. Is there a reason why you expect it to see in 0.1.0?
I hope this is available in the 0.4.1 version as well.

Regards,
Neetu

**Comment by eaglerainbow** [comment]
> I was following this [document](https://github.com/promregator/promregator/blob/master/docs/quickstart-docker.md) so I was using 0.1.0. I will definitely try the 0.4.1 version.

Mmhm.. well, that's an oldish statement in the documentation, which I have fixed with the commits mentioned above. 

> I hope this is available in the 0.4.1 version as well.

Yes. Promregator is using [semantic versioning](https://semver.org/). Special cases are at least mentioned in our [release notes](https://github.com/promregator/promregator/releases) of the corresponding version. 

I hope this will fix your issue. In case that this is not the case, please feel free to reopen this issue here (in case that it's again about using `cf.skipSslValidation`, or create a new issue.
